wd_aead.c

```diff
@@ -520,6 +523,14 @@ int wd_do_aead_sync(handle_t h_sess, struct wd_aead_req *req)
 		goto err_out;
 	}
 
+	if (req->in_bytes >= POLL_SIZE) {
+		ret = wd_ctx_wait(ctx->ctx, POLL_TIME);
+		if (ret < 0) {
+			WD_ERR("wd ctx wait err(%d)!\n", ret);
+			goto err_out;
+		}
+	}
+
```

Comment:
Do you mean that adopt polling if source buffer is larger than POLL_SIZE limitation? Otherwise, the source buffer is small, and use busy-loop wait to receive output from hardware. Is it right?

Reply:
If req in_bytes is bigger，it means need more time for hardware to do one task, I pick this POLL_SIZE from 920 test data, we use poll wait instead of busy-loop to reduce CPU usage.But this function need kernel support poll, do you think it is better to support it after mainline upload poll？

Reply:
While your patch is applied, will it improve performance?
From current data, synchronous mode gain much more performance on asynchronous mode. If you could observe the same result, I suggest hold it on until the performance on asynchronous mode is improved.

Reply:
This patch will not improve performance,its main purpose was  to reduce cpu usage for sync mode.but I can still try to use it for async mode, i will close this request temporarily